src/operator/nn/mkldnn/mkldnn_fully_connected-inl.h

```diff
@@ -80,6 +86,67 @@ struct MKLDNNFCFullParam {
   std::vector<float> output_scales = {0.0f};
 };
 
+#if MXNET_USE_MKLDNN == 1
```

Comment:
This #if is redundant, please have a look at the line: 30 (we don't need to have double-check)

Reply:
ok